aten/src/ATen/native/native_functions.yaml

```diff
@@ -4293,8 +4347,6 @@
   variants: function, method
 
 - func: square.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
-  dispatch:
```

Comment:
Actually, what's the reason you removed this?

Reply:
Sorry, should have mentioned before. I'm getting `RuntimeError: derivative for aten::square is not implemented` while adding support for `square`. So, I thought I'll take a look at this in a separate PR if that's alright?

Reply:
Hm, but this PR doesn't add support square if I read it correctly. Why remove the dispatch for CPU and CUDA here?

Reply:
Oh, my bad. Didn't mean to remove it. I'll bring it back.

Reply:
Done. Sorry again, @cpuhrsch! :)